airflow/www/auth.py

```diff
@@ -46,7 +46,7 @@ def decorated(*args, **kwargs):
             )
             if appbuilder.sm.check_authorization(permissions, dag_id):
                 return func(*args, **kwargs)
-            elif not g.user.is_anonymous and not g.user.perms:
+            elif auth_manager.is_logged_in() and not g.user.perms:
```

Comment:
I suppose `g.user.perms` is something we'll touch too, but later on in the project ?

Reply:
Exactly. The plan is to remove `g.user` from the code, but as you said, step by step :)